### PR TITLE
Utility startup details added

### PR DESCRIPTION
**Pull request by nijugeorge173**
# Description

Following are details of PR : 
--------------------------

1) Added documentation detailing the file location for config.yml ( ./configs/config.yml)
2) Environment variable for license key ('NEW_RELIC_LICENSE_KEY') should be added during setup. When missing the utility fails with error that license key needs to be of length 40. After adding the environment variable the utility starts up properly.

### CODE REVIEW

**Review by sdewitt-newrelic — CHANGES_REQUESTED** [review]
@nijugeorge173 sorry about that, though I had clicked submit review already but missed that step.  Please see requested changes.

File: README.md [hunk 1]
```diff
@@ -256,8 +256,8 @@ or as an AWS Lambda function.
    for your platform
 2. Extract the archive to a new directory
 3. Create a new configuration file from
-   [the sample configuration file](configs/config.sample.yml)
-4. Set the appropriate environment variables
+   [the sample configuration file](configs/config.sample.yml).Place the configuration file inside 'configs' folder created at same location at CMDB utility executable
+4. Set the appropriate environment variables. Environment variable 'NEW_RELIC_LICENSE_KEY' is mandatory.
```
[the remark this answers is not in the document]
Comment (nijugeorge173) [reply README.md:260]: It gave a error on InterOp stating that license key needs to be of length 40 and the utility did not run.

[the remark this answers is not in the document]
Comment (nijugeorge173) [reply README.md:260]: Added the PR description

File: README.md [hunk 2]
```diff
@@ -256,8 +256,8 @@ or as an AWS Lambda function.
    for your platform
 2. Extract the archive to a new directory
 3. Create a new configuration file from
-   [the sample configuration file](configs/config.sample.yml)
-4. Set the appropriate environment variables
+   [the sample configuration file](configs/config.sample.yml).Place the configuration file inside 'configs' folder created at same location at CMDB utility executable
```
[the remark this answers is not in the document]
Comment (nijugeorge173) [reply README.md]: Pushed

**Review by jbeveland27 — APPROVED** [review]

**Pull request merged**